dbms/src/TestUtils/tests/gtest_column_generator.cpp

```diff
@@ -0,0 +1,37 @@
+// Copyright 2022 PingCAP, Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <TestUtils/ColumnGenerator.h>
+#include <TestUtils/TiFlashTestBasic.h>
+
+namespace DB
+{
+namespace tests
+{
+
+TEST(TestColumnGenerator, run)
```

Comment:
I think there is no need to test

Reply:
> I think there is no need to test

Maybe it will be used in the future. The code is not used in the project, so I just use the test to ensure that the function will not crash

Reply:
There is no need to consider a lot for "future". The logical of this case is simple, local test is enough. Unit test give pressure to ci.